packages/typeset/typeset.0.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+synopsis: "An embedded DSL for defining source code pretty printers"
+maintainer: "Soren Norbaek <sorennorbaek@gmail.com>"
+authors: "Soren Norbaek <sorennorbaek@gmail.com>"
+license: "MIT"
+homepage: "https://github.com/soren-n/typeset-ocaml"
+bug-reports: "https://github.com/soren-n/typeset-ocaml/issues"
+dev-repo: "git+https://github.com/soren-n/typeset-ocaml.git"
+build: [
+  "dune" "build" "-p" name "-j" jobs "@install"
+  "@runtest" {with-test}
+]
+depends: [
+  "ocaml" {>= "4.08" & < "4.13"}
```

Comment:
I'm curious, what is the reason behind the `< "4.13"` constraint? If it is "just in case" don't worry about it, we add the missing constraints ourselves if needed

Reply:
yeah, it was because I saw that some of the ci checks were failling wrt 4.13, wasn't sure if all checks must be green for the pr to go through, so I made the addition. I can remove it if you recommend.

Reply:
CI doesn't have to be green to merge. OCaml 4.13 is currently not even branched off of trunk so this a lot more unstable. There was a temporary issue that made dune fail to compile but it's fixed now.